src/style.css

```
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin: 0 8px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form-field {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input-icon {
+  width: 100%;
+}
+
+.input-icon img {
+  position: absolute;
+}
+
+.icon--big {
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.icon--small {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  outline: none;
+}
+
+.big--bar {
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+  font-size: 16px;
+  font-weight: 800;
+}
+
+.small--bar {
+  padding: 10px 0 10px 33px;
+  line-height: 19px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+input:hover {
```

```suggestion
.input:hover {
```
use classes instead of tags